addons/artillerytables/functions/fnc_turretPFEH.sqf

```diff
@@ -31,6 +31,15 @@ if (isNull (uiNamespace getVariable [QGVAR(display), displayNull])) then {
 
 private _ctrlGroup = (uiNamespace getVariable [QGVAR(display), displayNull]) displayCtrl 1000;
 
+if (_elevationMode == 0) then {
+    // For turrets not using an elevation mode (i.e. mouse controls elevation instead of pageUp/Down)
+    // Show the info 1 step below, so it doesn't block vanilla ranging (discreteDistance)
+    // vanilla zeroing will have a negative effect, so it's best to range as low as possible
+    private _y = 3.5 * ((((safeZoneW / safeZoneH) min 1.2) / 1.2) / 25) + (profileNamespace getVariable ['IGUI_GRID_WEAPON_Y', (safeZoneY + 0.5 * ((((safeZoneW / safeZoneH) min 1.2) / 1.2) / 25))]);
```

Comment:
```suggestion
    private _safeZoneX = (((safeZoneW / safeZoneH) min 1.2) / 1.2) / 25;
    private _y = 3.5 * _safeZoneX + (profileNamespace getVariable ['IGUI_GRID_WEAPON_Y', safeZoneY + 0.5 * _safeZoneX]);
```
_(Ain't sure it's actually the X coordinate, in my defence.)_

Reply:
Changed from `_safeZoneX` to `_safeZoneGrid`
not sure what to call it but it's not a x/width